apps/patient/src/utils/text.json

```diff
@@ -37,8 +37,12 @@
     "heading": "Filling your prescriptions",
     "subheading": "After your prescriptions are sent to a pharmacy, we'll send you updates on their status.",
     "pickup": "Pick Up:",
+    "chat": {
+      "prompt": "At the pharmacy? Let us know if you're having any issues.",
+      "cta": "Text us now"
+    },
     "SENT": {
-      "title": "Sent to pharmacy",
+      "title": "Sent to Pharmacy",
       "description": "Your prescription fills have been sent to the pharmacy."
```

Comment:
Unrelated, this copy seems a little strange

Reply:
Just matching casing on status titles